packages/carbon-web-components/src/components/layer/layer.stories.ts

```diff
@@ -70,37 +79,30 @@ export const UseLayer = () => {
     <style>
       ${styles}
     </style>
-  `;
+  `
+  }
 };
 
-UseLayer.storyName = 'useLayer';
-
-export const Playground = (args) => {
-  const { level } = args?.[`${prefix}-layer-playground`] ?? {};
-  return html`
+export const Playground = {
+  args: defaultArgs,
+  argTypes: controls,
+  render: ({level}) => html`
```

Comment:
Here again, should be be bringing in the parameters? eg.
```suggestion
  parameters: {
    percy: {
      skip: true,
    },
  },
  render: ({level}) => html`
```